### ISSUE TITLE
Trigger update on initial load

### ISSUE THREAD

**Issue by eric-burel** [hoppula/react-router-redux-params#3]
Hi,

For the moment, when you load the store for the first time, the `syncParams` callback is not triggered. That's because it relies on `history.listen`, which is triggered only on change.

So opening the app on a specific route or refreshing the app won't correctly instantiate the param object, we will need to wait for a route change.

Here is how I tweaked the lib to fire an update on the initial load:

```
const updateParams = (store, routesArray) => (location) => {
  matchRoutes(routesArray, location, (error, state) => {
    if (!error) {
      store.dispatch({
        type: UPDATE_LOCATION_WITH_PARAMS,
        payload: {
          locationBeforeTransitions: location,
          params: state ? state.params : {}
        },
          // extend to work with electron-redux
        meta: {
          scope: 'local'
        }
      });
    }
  });
};
export function syncParams(store, routes, history) {
  const routesArray = createRoutes(routes);
  const updateCallback = updateParams(store, routesArray)
  ;
  // call the callback once to sync initial params
  updateCallback(history.getCurrentLocation());
  return history.listen(updateCallback);
}
```

I can't PR because I modified a lot of stuff to get it working on the correct versions of `react-router` (plus I seriously lack the time^^), but at least this is a working solution.

**Comment by hoppula** [comment]
Oh, good catch. Thanks for this, I'll try to incorporate your changes as soon as I get some time to work on this :+1: 

I saw that you've added `meta: { scope: 'local'}` to the dispatched action, probably this lib could also take an optional custom object that's always extended to default action?

**Comment by eric-burel** [comment]
Yep why not. Even better, you could simply let the user pass a custom action creator that takes `location` and `state` as entry params.

Anyway this lib works great and solves the strong limitations of the current `react-router-redux` version, thanks for you work.

**Comment by hoppula** [comment]
Just released 2.0.0 with your proposed changes, thanks again! 👍
